ecl/eclccserver/eclccserver.cpp

```diff
@@ -347,6 +347,13 @@ class EclccCompileThread : implements IPooledThread, implements IErrorReporter,
         eclccCmd.appendf(" -platform=%s", target);
         eclccCmd.appendf(" --component=%s", queryStatisticsComponentName());
 
+        IPropertyTree *costs = queryCostsConfiguration();
```

Comment:
@shamser this is likely to change in a PR I am nearly finished with - eclcc will be passed the eclccserver configuration file, so it can read the values directly.